control/control_validator/src/control_validator.cpp

```diff
@@ -46,14 +46,11 @@ ControlValidator::ControlValidator(const rclcpp::NodeOptions & options)
 
   setupParameters();
 
-  if (publish_diag_) {
-    setupDiag();
-  }
+  setupDiag();
```

Comment:
Why was this `publish_diag_` parameter removed?

Reply:
Please have a log to the [sheet](https://docs.google.com/spreadsheets/d/1z6RTyKQuaERNli5b-iCdj1ottrK84h_RUKbKmkc7YxU/edit#gid=0), warnings tab, row 22

Reply:
Thanks for the clarification.

I also agree we can publish this HW_ID by default since it will complain otherwise.

@soblin @kyoichi-sugahara @takayuki5168 Do you agree? Can we remove this parameter?

Reply:
@ahmeddesokyebrahim @xmfcx 
Sorry for late reply and thank you so much for investigating the problem.
Yes, you cen remove the paramter :+1:

Reply:
cc. @mitsudome-r